covidsafe.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.example.corona.comms";
+option java_outer_classname = "TestProto";
+option objc_class_prefix = "CC";
+
+package corona;
+
+service Registration {
+	rpc registerUser (RegistrationInfo) returns (Registered) {}
+}
+
+service Logging {
+	rpc sendInfectedLogs (stream Log) returns (AddedLogs) {}
+	rpc getBLTContactLogs (stream Key) returns (stream BLTContactLog) {}
+}
+
+message RegistrationInfo {
+	string phone = 1;
+	string key = 2;
+}
+
+message Registered {
+	bool success = 1;
+}
+
+message InfectedUserData {
+	string phone = 1;
+	string key = 2;
+	string dob = 3;
```

Comment:
We should probably make this `google.protobuf.Timestamp` type or `int64` datatype.